db/seed/001_development_seed.sql

```diff
@@ -244,6 +244,17 @@ BEGIN TRY
     INSERT INTO Configuration.KeyValue (Name, Value, Text, Description, Created, CreatedBy, LastModified, LastModifiedBy) VALUES (N'QaReturnReason', N'Information conflicts with CATS', N'Information conflicts with CATS', N'A QA return reason', @Created, @CreatedUserId, null, null);
     INSERT INTO Configuration.KeyValue (Name, Value, Text, Description, Created, CreatedBy, LastModified, LastModifiedBy) VALUES (N'QaReturnReason', N'Other', N'Other', N'A QA return reason', @Created, @CreatedUserId, null, null);
 
+    -- Education Level
+    INSERT INTO Configuration.KeyValue (Name, Value, Text, Description, Created, CreatedBy, LastModified, LastModifiedBy) VALUES (N'EducationLevel', N'Entry', N'Entry', N'An Education Level', @Created, @CreatedUserId, null, null);
+    INSERT INTO Configuration.KeyValue (Name, Value, Text, Description, Created, CreatedBy, LastModified, LastModifiedBy) VALUES (N'EducationLevel', N'Level 1', N'Level 1', N'An Education Level', @Created, @CreatedUserId, null, null);
+    INSERT INTO Configuration.KeyValue (Name, Value, Text, Description, Created, CreatedBy, LastModified, LastModifiedBy) VALUES (N'EducationLevel', N'Level 2', N'Level 2', N'An Education Level', @Created, @CreatedUserId, null, null);
+    INSERT INTO Configuration.KeyValue (Name, Value, Text, Description, Created, CreatedBy, LastModified, LastModifiedBy) VALUES (N'EducationLevel', N'Level 3', N'Level 3', N'An Education Level', @Created, @CreatedUserId, null, null);
+    INSERT INTO Configuration.KeyValue (Name, Value, Text, Description, Created, CreatedBy, LastModified, LastModifiedBy) VALUES (N'EducationLevel', N'Level 4', N'Level 4', N'An Education Level', @Created, @CreatedUserId, null, null);
+    INSERT INTO Configuration.KeyValue (Name, Value, Text, Description, Created, CreatedBy, LastModified, LastModifiedBy) VALUES (N'EducationLevel', N'Level 5', N'Level 5', N'An Education Level', @Created, @CreatedUserId, null, null);
+    INSERT INTO Configuration.KeyValue (Name, Value, Text, Description, Created, CreatedBy, LastModified, LastModifiedBy) VALUES (N'EducationLevel', N'Level 6', N'Level 6', N'An Education Level', @Created, @CreatedUserId, null, null);
+    INSERT INTO Configuration.KeyValue (Name, Value, Text, Description, Created, CreatedBy, LastModified, LastModifiedBy) VALUES (N'EducationLevel', N'Level 7', N'Level 7', N'An Education Level', @Created, @CreatedUserId, null, null);
+    INSERT INTO Configuration.KeyValue (Name, Value, Text, Description, Created, CreatedBy, LastModified, LastModifiedBy) VALUES (N'EducationLevel', N'Level 8', N'Level 8', N'An Education Level', @Created, @CreatedUserId, null, null);
+
```

Comment:
Can I suggest adding a description to each level for clarity? These were taken from the UK RQF - Regulated Qualifications Framework (RQF)

```suggestion
    INSERT INTO Configuration.KeyValue (Name, Value, Text, Description, Created, CreatedBy, LastModified, LastModifiedBy) VALUES (N'EducationLevel', N'Entry', N'Entry Level', N'An Education Level', @Created, @CreatedUserId, null, null);
    INSERT INTO Configuration.KeyValue (Name, Value, Text, Description, Created, CreatedBy, LastModified, LastModifiedBy) VALUES (N'EducationLevel', N'Level 1', N'Level 1 (GCSE grades D-G or 1-3, Foundation Diplomas, NVQ Level 1)', @Created, @CreatedUserId, null, null);
    INSERT INTO Configuration.KeyValue (Name, Value, Text, Description, Created, CreatedBy, LastModified, LastModifiedBy) VALUES (N'EducationLevel', N'Level 2', N'Level 2 (GCSE grades A*-C or 4-9, Intermediate Apprenticeships, NVQ Level 2)', N'An Education Level', @Created, @CreatedUserId, null, null);
    INSERT INTO Configuration.KeyValue (Name, Value, Text, Description, Created, CreatedBy, LastModified, LastModifiedBy) VALUES (N'EducationLevel', N'Level 3', N'Level 3 (A Levels, T Levels, BTEC Nationals, Advanced Apprenticeships)', N'An Education Level', @Created, @CreatedUserId, null, null);
    INSERT INTO Configuration.KeyValue (Name, Value, Text, Description, Created, CreatedBy, LastModified, LastModifiedBy) VALUES (N'EducationLevel', N'Level 4', N'Level 4 (Certificates of Higher Education, HNC, Level 4 NVQ)', N'An Education Level', @Created, @CreatedUserId, null, null);
    INSERT INTO Configuration.KeyValue (Name, Value, Text, Description, Created, CreatedBy, LastModified, LastModifiedBy) VALUES (N'EducationLevel', N'Level 5', N'Level 5 (Diplomas of Higher Education, Foundation Degrees, HND)', N'An Education Level', @Created, @CreatedUserId, null, null);
    INSERT INTO Configuration.KeyValue (Name, Value, Text, Description, Created, CreatedBy, LastModified, LastModifiedBy) VALUES (N'EducationLevel', N'Level 6', N'Level 6 (Bachelors Degree, Degree Apprenticeships, Level 6 Diplomas)', N'An Education Level', @Created, @CreatedUserId, null, null);
    INSERT INTO Configuration.KeyValue (Name, Value, Text, Description, Created, CreatedBy, LastModified, LastModifiedBy) VALUES (N'EducationLevel', N'Level 7', N'Level 7 (Masters Degree, Postgraduate Diplomas, Level 7 NVQs)', N'An Education Level', @Created, @CreatedUserId, null, null);
    INSERT INTO Configuration.KeyValue (Name, Value, Text, Description, Created, CreatedBy, LastModified, LastModifiedBy) VALUES (N'EducationLevel', N'Level 8', N'Level 8 (Doctorates, e.g., PhD, Professional Doctorates)', N'An Education Level', @Created, @CreatedUserId, null, null);

```